js/app.js

```diff
@@ -95,3 +95,22 @@ createTable();
 for (var i = 0; i < allStores.length; i++) {
   allStores[i].generateTableRow();
 }
+
+var elStoreForm = document.getElementById('newStoreForm');
+
+function submitButton(event) {
+  event.preventDefault();
+  var newStoreForm = event.target;
+  var strLoc = event.target.storeLocation.value;
+  var minCst = Math.round(event.target.minimumCustomers.value);
+  var maxCst = Math.round(event.target.maximumCustomers.value);
+  var avgCook = event.target.averageCookies.value;
+  if (minCst > maxCst) {
+    alert('Please double check the number values for accuracy and re-submit the form. Thank you.');
```

Comment:
An alert is fine, but it is always without exception a nuisance for the user. Instead, you might want to consider adding a div in the HTML whose only purpose is to display an error message to the user if there is one to be shown.